src/server.tsx

```diff
@@ -420,9 +420,6 @@ app.get(
             const ampVariantAssignments = getAmpVariantAssignments(req);
             const response = await ampEpic(ampVariantAssignments, countryCode);
 
-            // The cache key in fastly is the X-GU-GeoIP-Country-Code header
-            res.setHeader('Surrogate-Control', 'max-age=120');
-            res.setHeader('Cache-Control', 'max-age=60');
```

Comment:
We can't cache this endpoint as each user is sending user specific data through the query string!

Reply:
👍 this will increase load, but these requests are only made when the user scrolls half way down the page so I don't expect a huge impact